src/gui/src/AboutDialog.cpp

```diff
@@ -49,6 +49,8 @@ AboutDialog::AboutDialog(QWidget* parent)
 	about += tr( "<b>Project page</b><br>");
 	about += "http://sourceforge.net/projects/hydrogen<br><br>";
 	about += tr( "<b>Mailing lists</b><br>");
+	about += "http://lists.sourceforge.net/lists/listinfo/hydrogen-users<br>";
+	about += "http://lists.sourceforge.net/lists/listinfo/hydrogen-announce<br>";
```

Comment:
Maybe a part of the "dead" situation is coming from the fact they are not advertised ? I'm thinking especially of the use ML here.